app/src/main/java/fr/free/nrw/commons/media/MediaInterface.java

```diff
@@ -0,0 +1,15 @@
+package fr.free.nrw.commons.media;
+
+import org.wikipedia.dataclient.mwapi.MwQueryResponse;
+
+import io.reactivex.Observable;
+import retrofit2.http.GET;
+import retrofit2.http.Query;
+
+public interface MediaInterface {
```

Comment:
Quick javadoc explaining what is the goal of this interface, when to use it? :-)

Reply:
Yes, will add it. There are more issues with the PR as of now. So I have kept it as a draft PR. :)

Reply:
Done. :)